actors/miner/src/lib.rs

```diff
@@ -514,7 +514,7 @@ impl Actor {
 
             // Verify the chain commit randomness
             let comm_rand = rt.get_randomness_from_tickets(
-                DomainSeparationTag::PoStChainCommit,
+                DomainSeparationTag::PoStChainCommit as i64,
```

Comment:
I know this is annoying, but let's break this out into a separate PR and land on master first. Otherwise, we'll diverge from master which will be annoying to maintain.

Reply:
mentioned verbally, but for posterity: we are going with seperate "user" randomness functions that dont affect the types of get_randomness_from_tickets so we dont have any conflicts